src/class-fields/computed-name-empty-var-empty-function.case

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2016 the V8 project authors, 2017 Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: Computed name var not initialized, empty function
+info: |
+  1.1 New Productions
+
+  [...]
+
+  FieldDefinitionList [Yield, Await]:
+    FieldDefinition [?Yield, ?Await]
+    FieldDefinitionList  [?Yield, ?Await], FieldDefinition [?Yield, ?Await]
+template: default
+includes: [propertyHelper.js]
+---*/
+
+//- body
+  ['a']; b(){}
+
+//- assertions
+  assert.sameValue(this.a, undefined);
+  verifyEnumerable(this, "a");
+  verifyWritable(this, "a");
+  verifyConfigurable(this, "a");
+
+  assert.sameValue(typeof Object.getPrototypeOf(this).b, "function");
+  verifyNotEnumerable(Object.getPrototypeOf(this), "b");
+  verifyConfigurable(Object.getPrototypeOf(this), "b");
```

Comment:
overcalling Object.getPrototypeOf(this) can really be improved here as Rick suggested.

It does not hurt checking `this` has no own property `a` or `b` and get the property from the prototype of the instance.

For this case, I suggest:

```
class Base {}
class C extends Base {
  ['a']; b(){ return 42; }

  constructor() {
    super();

    assert.sameValue(Object.hasOwnProperty.call(this, 'a'), true);
    assert.sameValue(Object.hasOwnProperty.call(this, 'b'), false); // redundant, already covered, but shrugs

    const proto = Object.getPrototypeOf(this);

    assert.sameValue(this.a, undefined);
    assert.sameValue(this.b(), 42);

    verifyProperty(this, 'a', {
      value: undefined,
      enumerable: true,
      writable: true,
      configurable: true,
    });
  }
}

new C();
```

Remember we are not testing the property values of method definitions here, these are covered in other files. So I wouldn't bother.